pydecred/rpc.py

```diff
@@ -60,12 +60,183 @@ def getBlockchainInfo(self):
 
+    def searchRawTransactions(
+        self,
+        address,
+        verbose=1,
+        skip=0,
+        count=100,
+        vinextra=0,
+        reverse=False,
+        filteraddrs=[],
+    ):
+        """
+        Returns raw data for transactions involving the passed address. Returned
+        transactions are pulled from both the database, and transactions currently
+        in the mempool. Transactions pulled from the mempool will have the
+        'confirmations' field set to 0. Usage of this RPC requires the optional
+        --addrindex flag to be activated, otherwise all responses will simply
+        return with an error stating the address index has not yet been built.
+        Similarly, until the address index has caught up with the current best
+        height, all requests will return an error response in order to avoid
+        serving stale data.
+
+        Args:
+            address (str): The Decred address to search for
+            verbose (int): Optional. default=1. Specifies the transaction is returned as a JSON
+                object instead of hex-encoded string.
+            skip (int): Optional. Default=0. The number of leading transactions to leave out of the
+                final response.
+            count (int): Optional. Default=100. The maximum number of transactions to return.
+            vinextra (int): Optional. Default=0. Specify that extra data from previous output will be
+                returned in vin.
+            reverse (bool): Optional. Default=False. Specifies that the transactions should be returned
+                in reverse chronological order.
+            filteraddrs (list(str)): Optional. Default=[]. Only inputs or outputs with matching
+                address will be returned.
+
+        Returns:
+            list(SearchRawTransactionResult): The SearchRawTransactionResults.
+        """
+        return [
+            SearchRawTransactionsResult.parse(rawTx)
+            for rawTx in self.call(
+                "searchrawtransactions",
+                address,
+                verbose,
+                skip,
+                count,
+                vinextra,
+                reverse,
+                filteraddrs,
+            )
+        ]
+
+    def sendRawTransaction(self, msgTx, allowhighfees=False):
+        """
+        Submits the serialized, hex-encoded transaction to the local peer and
+        relays it to the network.
+
+        Args:
+            msgTx (object): msgtx.MsgTx signed transaction.
+            allowhighfees (bool): Optional. Default=False. Whether or not to allow insanely high fees
+                (dcrd does not yet implement this parameter, so it has no effect).
+
+        Returns:
+            bytes-like: The hash of the transaction.
+        """
+        txid = self.call("sendrawtransaction", msgTx.txHex(), allowhighfees)
+        return reversed(ByteArray(txid))
+
+    def setGenerate(self, generate, genproclimit=-1):
```

Comment:
~~genproclimit~~ -> genProcLimit

Or do it the *dcrd/rpcclient* way

`func (c *Client) SetGenerate(enable bool, numCPUs int) error`

Reply:
numCPUs is more understandable. Thank you.